<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkConnection</class>
 <widget class="QWidget" name="NetworkConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="passwordEdit_port">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit_IPaddress">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
  </widget>
  <widget class="QLabel" name="Designedby">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Designed by Young Inc Â© 2021 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonOk">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}</string>
   </property>
   <property name="text">
    <string>Cancle</string>
   </property>
  </widget>
  <widget class="QLabel" name="BK">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>560</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>560</width>
     <height>500</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Background/BK1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>BK</zorder>
  <zorder>passwordEdit_port</zorder>
  <zorder>passwordEdit_IPaddress</zorder>
  <zorder>Designedby</zorder>
  <zorder>pButtonOk</zorder>
  <zorder>pButtonCancel</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
